ramls/holdings-storage.raml

```diff
@@ -28,17 +30,23 @@ resourceTypes:
     displayName: Holdings
     type:
       collection:
-        exampleCollection: !include  examples/holdingsrecords_get.json
-        schemaCollection: holdingsRecords
-        schemaItem: holdingsRecord
-        exampleItem: !include  examples/holdingsrecord_get.json
+        exampleCollection: !include examples/holdings-storage/holdingsRecords_get.json
+        schemaCollection: holdingsRecordViews
+        schemaItem: holdingsRecordView
+        exampleItem: !include examples/holdings-storage/holdingsRecord_get.json
     get:
       is: [pageable,
            searchable: {description: "by instance ID (using CQL)",
                         example: "instanceId=\"2b94c631-fca9-4892-a730-03ee529ffe2a\""},
           ]
     post:
       is: [validate]
+      body:
+        application/json:
+          type: holdingsRecord
+          example:
+            strict: false
+            value: !include examples/holdings-storage/holdingsRecord_get.json
```

Comment:
Probably should be holdingsRecord_post.json or renamed to holdingsRecord.json if they are similar